File: csharp/Services/OutputFormatter.cs
using System;
using System.Globalization;
using PhpSales.Entities;

namespace PhpSales.Services
{
    public static class OutputFormatter
    {
        private static string FormatItem(Item item)
        {
            return $"{item.Quantity} {item.Name}: {item.TaxedPrice.ToString("F2", CultureInfo.InvariantCulture)}";
        }

        public static void PrintCart(Cart cart)
        {
            foreach (var item in cart.GetItems())
            {
                string formatted = FormatItem(item);
                Console.WriteLine(formatted + "<br />\n");
            }

            Console.WriteLine($"Sales Taxes: {cart.GetTax().ToString("F2", CultureInfo.InvariantCulture)}<br />\n");
            Console.WriteLine($"Total: {cart.GetTotal().ToString("F2", CultureInfo.InvariantCulture)}<br />\n");
        }
    }
}

File: src/Services/OutputFormatter.php
<?php
namespace src\Services;
use src\Entities\Item;

class OutputFormatter
{

    private static function formatItem($item): string
    {
        return $item->quantity.' '.$item->name.': '.
            number_format($item->taxedPrice, 2, '.', '');
    }

    public static function printCart($cart): void
    {
        foreach ($cart->getItems() as $item) {
            $formattedOutput = self::formatItem($item);
            echo $formattedOutput . "<br />\n";
        }

        echo "Sales Taxes: " . number_format($cart->getTax(), 2, '.', ''). "<br />\n";
        echo "Total: " . number_format($cart->getTotal(), 2, '.', '') . "<br />\n";
    }
}